python-sdk/src/astro/sql/operators/cleanup.py

```diff
@@ -224,17 +224,38 @@ def resolve_tables_from_tasks(  # noqa: C901
         for task in tasks:
             try:
                 if isinstance(task, OPERATOR_CLASSES_WITH_TABLE_OUTPUT):
-                    t = task.output.resolve(context)
-                    if isinstance(t, BaseTable):
-                        res.append(t)
+                    try:
+                        # works on airflow version >= 2.4.0
```

Comment:
If we now the exact airflow version, can't we decide based on this which path to go?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1385/commits/230add9e1d087f9fa6b1ae007e684aa8360db1a6)